### PR TITLE
Changed component for Hazard ID & changed titles

### LINKED ISSUE #214: Step 3-4 replace form dropdown with another component
We need a better UI widget for selecting hazard (and datasets). E.g. if we have 200 hazards then this dropdown will be too long to scroll
image

### PR DESCRIPTION
**Pull request by ylyangtw**
Changed 2 things.
1. Changed the component of Hazard ID to autocomplete. This way you can type to search in the menu.
2. Changed the titles to be the same as the salt lake city playbook.

### CODE REVIEW

File: src/datawolf.config.js [hunk 1]
```diff
@@ -518,7 +518,7 @@ const datawolfWorkflowConfig =
 					]
 				},
 				{
-					"title": "hazard_type",
+					"title": "Hazard Type",
```
Comment (longshuicy) [inline src/datawolf.config.js:521]: Good catch here! I forgot this is the pretty name that get showed on the form

File: src/components/styledComponents/IncoreSelect.jsx [hunk 2]
```diff
@@ -30,3 +31,41 @@ export const IncoreMenuItem = styled(MenuItem)({
 	lineHeight: "18px",
 	backgroundColor: "#FFFFFF"
 });
+
+export const IncoreAutocomplete = styled(Autocomplete)({
```
Comment (longshuicy) [inline src/components/styledComponents/IncoreSelect.jsx]: Could this be its own styled component cuz it's not quite select?

File: src/components/styledComponents/IncoreSelect.jsx [hunk 3]
```diff
@@ -30,3 +31,41 @@ export const IncoreMenuItem = styled(MenuItem)({
...
+	position: "relative",
+	"&:before, &:after": {
+		borderColor: "#00619D",
+	},
+	"& .MuiSelect-select:focus": {
+		backgroundColor: "#FFFFFF!important"
+	},
+});
+
+export const IncoreTextField = styled(TextField)({
```
Comment (longshuicy) [inline src/components/styledComponents/IncoreSelect.jsx]: Is this a special TextField that required for autocomplete? If so can you put it in the same autocomplete component and rename it to be more descriptive? 

I remember we have the IncoreTextField styled component already but I could be wrong...

File: src/components/styledComponents/IncoreSelect.jsx [hunk 4]
```diff
@@ -30,3 +31,41 @@ export const IncoreMenuItem = styled(MenuItem)({
...
+	lineHeight: "18px",
+	backgroundColor: "#FFFFFF",
+	position: "relative",
+	"&:before, &:after": {
+		borderColor: "#00619D",
+	},
+	"& .MuiSelect-select:focus": {
+		backgroundColor: "#FFFFFF!important"
+	},
+});
```
Comment (longshuicy) [inline src/components/styledComponents/IncoreSelect.jsx]: Is it possible to add a border to the autocomplete to match other fields? 
image

Comment (ylyangtw) [reply src/components/styledComponents/IncoreSelect.jsx]: Added! 
image

File: src/components/styledComponents/IncoreSelect.jsx (hunk 2 above)
Comment (ylyangtw) [reply src/components/styledComponents/IncoreSelect.jsx]: Removed IncoreAutocomplete from IncoreSelect.jsx and made one for its own

File: src/components/styledComponents/IncoreSelect.jsx (hunk 3 above)
Comment (ylyangtw) [reply src/components/styledComponents/IncoreSelect.jsx]: Simply used IncoreTextField that we already have since it also works for Autocomplete

**Review by longshuicy — APPROVED** [review]
Looks good. Can you rebase/merge main back into your branch?
The latest version is 0.5.8 and could you check the changelog

**Pull request merged**